File: src/main/java/ru/hlebozavod28/massak/domain/Workplace.java
package ru.hlebozavod28.massak.domain;

import lombok.Data;

import javax.persistence.*;
import java.util.List;

@Data
@Entity
@Table(name = "workplaces", schema = "workplaces")
public class Workplace {
    @Id
    @GeneratedValue
    long id;
    @Column(name = "workplace_name")
    String WorkPlaceName;
    @ManyToMany(mappedBy = "workplaces")
    List<Scale> scales;
}

File: src/main/java/ru/hlebozavod28/massak/domain/Workplace.kt
package ru.hlebozavod28.massak.domain

import javax.persistence.*

@Entity
@Table(name = "workplaces", schema = "workplaces")
class Workplace {
    @Id
    @GeneratedValue
    var id: Long = 0

    @Column(name = "workplace_name")
    var workPlaceName: String? = null

    @ManyToMany(mappedBy = "workplaces")
    lateinit var scales: MutableList<Scale>
}